falco/values.yaml

```diff
@@ -177,6 +177,13 @@ falco:
   # for up to 2 seconds without reaching the timeout.
 
   output_timeout: 2000
+  # Disable asynchronous CRI metadata fetching.
+  # This is useful to let the input event wait for the container metadata fetch
+  # to finish before moving forward. Async fetching, in some environments leads
+  # to empty fields for container metadata when the fetch is not fast enough to be
+  # completed asynchronously. This can have a performance penalty on your environment
+  # depending on the number of containers and the frequency at which they are created/started/stopped
+  disableCriAsync: false
```

Comment:
nit: should be `disableCRIAsync`

Reply:
Two letter abbreviations is both capital letters.
From three and up it's only the first in capital.

Couldn't find a good source but XMLHTTPRequest or xmlHttpRequest

Reply:
fair, renamed!